website/docs/d/ip_sla_monitoring_policy.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Tenant Policies"
+layout: "aci"
+page_title: "ACI: aci_ip_sla_monitoring_policy"
+sidebar_current: "docs-aci-data-source-ip_sla_monitoring_policy"
+description: |-
+  Data source for ACI IP SLA Monitoring Policy
+---
+
+# aci_ipsla_monitoring_policy #
+
+Data source for ACI IP SLA Monitoring Policy
+
+
+## API Information ##
+
+* `Class` - fvIPSLAMonitoringPol
+* `Distinguished Name` - uni/tn-{name}/ipslaMonitoringPol-{name}
+
+## GUI Information ##
+
+* `Location` - 
+
+
+
+## Example Usage ##
+
+```hcl
+data "aci_ip_sla_monitoring_policy" "example" {
+  tenant_dn  = aci_tenant.example.id
+  name       = "example"
+}
+```
+
+## Argument Reference ##
+
+* `tenant_dn` - (Required) Distinguished name of the parent Tenant object.
+* `name` - (Required) Name of the IP SLA Monitoring Policy object.
+
+## Attribute Reference ##
```

Comment:
Remove all teh Allowed values and required whenm or other conditions in the datasource file.